src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01-browser.md

```diff
@@ -19,25 +23,108 @@ export default function () {
 
+### options: returns <[Object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)>
+
+You can customize the creation of a new browser context using the following options.
+
+<!-- vale off -->
+
+| Option            | Type                   | Description |
+| ----------------- | ---------------------- | ----------- |
+| acceptDownloads   | boolean                | TODO |
+| bypassCSP         | boolean                | TODO |
+| colorScheme       | string                 | Emulates 'prefers-colors-scheme' media feature. It can be one of `"light"`, `"dark"`, `"no-preference"`. |
+| deviceScaleFactor | number                 | TODO |
+| extraHTTPHeaders  | object                 | TODO |
+| geolocation       | object                 | See [geolocation](#newcontext-geolocation). |
+| hasTouch          | boolean                | TODO |
+| httpCredentials   | object                 | Credentials for HTTP authentication. See: [httpCredentials](#newcontext-http-credentials). |
+| ignoreHTTPSErrors | boolean                | TODO |
+| isMobile          | boolean                | TODO |
+| javaScriptEnabled | boolean                | TODO |
+| locale            | string                 | TODO |
+| offline           | boolean                | TODO |
+| permissions       | Array                  | TODO |
+| reducedMotion     | string                 | TODO |
+| screen            | object                 | Emulates consistent window screen size available inside web page via window.screen. Is only used when the viewport is set. See: [screen](#newcontext-screen) |
+| timezoneID        | xx                     | TODO |
+| userAgent         | xx                     | TODO |
+| videosPath        | xx                     | TODO |
+| viewport          | xx                     | TODO |
+
+<!-- vale on -->
+
+### <a name="newcontext-geolocation">geolocation option</a>
+
+| Option    | Type   | Description |
+| --------- | :----- | ----------- |
+| latitude  | number | Latitude between -90 and 90. |
+| longitude | number | Longitude between -180 and 180. |
+| accuracy  | number | Non-negative accuracy value. Defaults to 0. |
+
+### <a name="newcontext-http-credentials">httpCredentials option</a>
+
+| Option   | Type   | Description |
+| -------- | ------ | ----------- |
+| username | string | Username to pass to the web browser for Basic HTTP Authentication. |
+| password | string | Password to pass to the web browser for Basic HTTP Authentication. |
+
+### <a name="newcontext-screen">screen option</a>
+
+| Option   | Type   | Description |
+| -------- | ------ | ----------- |
+| username | string | Username to pass to the web browser for Basic HTTP Authentication. |
+| password | string | Password to pass to the web browser for Basic HTTP Authentication. |
+
+
+Not yet implemented options:
+
+  * `baseURL`, `forcedColors`, `logger`, `proxy`, `recordHar`, `recordVideo`, `storageState`, `strictSelectors`, `videoSize`.
```

Comment:
We mentioned in #508 that we [probably don't need a "Missing APIs" section](https://github.com/grafana/k6-docs/pull/508#issuecomment-1050683827), and this feels similar.

So WDYT if we remove this and mention all options in the same table above, marking the missing ones with a red `x` that on hover shows "Not implemented", and if there's a GH issue for that specific option (or method), the same `x` links to it?

Reply:
Makes sense!